src/main/java/net/greatstart/dto/DtoProjectDescription.java

```diff
@@ -21,4 +22,53 @@
     private LocalDate dateStart;
     private LocalDate dateAdded;
     private byte[] image;
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 17 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)